src/VisualStudio/Core/Def/DocumentOutline/ExpansionOption.cs

```diff
@@ -16,16 +16,6 @@ internal enum ExpansionOption
         /// <summary>
         /// Collapse all nodes.
         /// </summary>
-        Collapse,
-        /// <summary>
-        /// Do not update node expansion (when we are generating a new UI model, we will use the default expansion).
-        /// This option is used when we generate a UI model using a newly created Document Symbol data model.
-        /// </summary>
-        NoChange,
-        /// <summary>
-        /// Apply current tree view expansion to nodes (if the caret is moved around, we want to preserve existing
-        /// expanded/collapsed node states as the underlying data model has not been updated).
-        /// </summary>
-        CurrentExpansion
+        Collapse
```

Comment:
this is now just a boolean.  remove?  (it will also simplify a bunch of consumptio code).